docker/rocal-with-tensorflow.dockerfile

```diff
@@ -56,14 +52,3 @@ WORKDIR $ROCAL_WORKSPACE
 RUN pip install --upgrade pip
 RUN git clone -b develop https://github.com/ROCm/rocAL && \
         mkdir build && cd build && cmake -D PYTHON_VERSION_SUGGESTED=${ROCAL_PYTHON_VERSION_SUGGESTED} ../rocAL && make -j8 && cmake --build . --target PyPackageInstall && make install
-
-WORKDIR /workspace
-
-#install tensorflow dependencies - Compile protos and Install TensorFlow Object Detection API.
-RUN cd ~/models/research/ && \
```

Comment:
if you need this for ```create_pet_tf_record.py```, our TF dockers already comes with ```models/research```. Only location change from ~/. Please recheck

Reply:
The code in create_pet_tf_record.py is adapted from this [file] (https://github.com/tensorflow/models/blob/master/research/object_detection/dataset_tools/create_pet_tf_record.py) so we no longer need this models package installation which causes GPU issues